frontend/styles/var/mixins.scss

```diff
@@ -175,6 +175,15 @@ $max-width: 2560px;
   }
 }
 
+@mixin copy-message {
```

Comment:
see comment re a `CopyMessage` component

Reply:
100% nice to have